contents/docs/api/post-only-endpoints.mdx

```diff
@@ -49,7 +49,7 @@ curl -v -L --header "Content-Type: application/json" -d '{
     }
     # ...
   ]
-}' https://app.posthog.com/capture/ 
+}' https://app.posthog.com/batch/ 
```

Comment:
@xvello I know you're looking at future endpoints. As far as I understand - "batch" events should also go to `e` right? Batch, capture etc. are all essentially just aliases to /e/

Reply:
`/batch` is an endpoint we listen on, and it's indeed routed to the same django endpoint as `/e`. The payload shape is not consistent with the batches that posthog-js sends to `/e` though, so let's not rush into consolidating this.

I'd say let's merge this PR now as it aligns the example with the documentation above. We'll have to support a long-tail of queries to these old endpoints anyway, no need to rush changes.